backend/src/payloads/AppUser.ts

```diff
@@ -261,3 +261,49 @@ export class AppUserInterviewAvailability {
   @IsDateString()
   end: string;
 }
+
+export class InducteePostRequest {
```

Comment:
Should put this payload class with the Request payload classes, probably after AppUserSignupRequest, to group  them up nicely together